src/ImpactX.cpp

```diff
@@ -117,4 +117,18 @@ namespace impactx
         } // end step loop
     }
 
+    void ImpactX::initElements ()
+    {
+        // initialize element sequence
+        myelements = { };
+
```

Comment:
The list is empty by default: this constructor is called in `ImpactX::ImpactX` implicitly when the class that owns this member variable is constructed.
Thus, we can skip it here :)
```suggestion
```

Reply:
Ah, you want to make sure they are cleared. That works :)